core/metacling/src/TCling.cxx

```diff
@@ -6919,14 +6943,87 @@ const char* TCling::GetClassSharedLibs(const char* cls)
    return 0;
 }
 
+/// This interface returns a list of dependent libraries in the form:
+/// lib libA.so libB.so libC.so. The first library is the library we are
+/// searching dependencies for.
+/// Note: In order to speed up the search, we display the dependencies of the
+/// libraries which are not yet loaded. For instance, if libB.so was loaded
+/// the list would contain: lib libA.so libC.so.
+static std::string GetSharedLibDepsSlow(std::string lib,
```

Comment:
Why not `const std::string &`?

Reply:
We have an assignment to `lib` a few lines below.